src/components/CvSearch/CvSearch.vue

```diff
@@ -33,20 +33,20 @@
       }}</label>
 
       <input
-        role="searchbox"
         :id="uid"
-        :class="`${carbonPrefix}--search-input`"
         v-bind="$attrs"
+        ref="elInput"
         v-model="internalSearchText"
-        @input="onInput"
+        role="searchbox"
+        :class="`${carbonPrefix}--search-input`"
         type="text"
-        ref="elInput"
         :placeholder="placeholder"
         :aria-labelledby="uid"
+        :disabled="disabled ? 'true' : undefined"
```

Comment:
following compatibility mode comment, might be that also here it should be changed to :disabled="disabled || null" ?